ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -324,7 +325,8 @@ - (void)updateCacheToItem:(ADTokenCacheItem *)cacheItem
         }
     }
     
-    AD_LOG_VERBOSE_F(@"Token cache store", correlationId, @"Storing access token for resource: %@", cacheItem.resource);
+    AD_LOG_VERBOSE(correlationId, nil, NO, @"Token cache store - Storing access token for resource: %@", cacheItem.resource);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 125 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)